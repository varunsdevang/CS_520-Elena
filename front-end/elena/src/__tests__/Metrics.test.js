import { render, screen } from '@testing-library/react';
import React from "react";
import renderer from "react-test-renderer";
import MetricTable from '../Components/Metrics';


describe("Metrics table snapshot should match", () => {
    it("Matches DOM Snapshot", () => {
      const domTree = renderer.create(< MetricTable/>).toJSON();
      expect(domTree).toMatchSnapshot();
    });
});