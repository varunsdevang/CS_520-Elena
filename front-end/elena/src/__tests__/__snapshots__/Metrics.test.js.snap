// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Metrics table snapshot should match Matches DOM Snapshot 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root routeinfo-table-container css-11xur9t-MuiPaper-root-MuiTableContainer-root"
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <h3>
       ROUTE INFORMATION 
    </h3>
  </div>
  <table
    aria-label="simple table"
    className="MuiTable-root css-rqglhn-MuiTable-root"
    role={null}
  >
    <tbody
      className="MuiTableBody-root css-apqrd9-MuiTableBody-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
        role={null}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          Distance Travelled: 
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          2.6 miles
        </td>
      </tr>
      <tr
        className="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
        role={null}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          Elevation Gain: 
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          125%
        </td>
      </tr>
      <tr
        className="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
        role={null}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          Est. Travel Time: 
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          7 mins
        </td>
      </tr>
      <tr
        className="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
        role={null}
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          Calories: 
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
        >
          234 cal/hr
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
